scripts/install_prerequisites.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash 
+
+
+sudo apt-get update
+ 
+sudo apt-get install git 
+
+sudo apt-get install unzip
+ sudo apt-get install default-jre
+
+sudo apt-get install curl python-setuptools python-dev python-pip 
```

Comment:
Maybe add the other things above to this line: sudo apt-get install git unzip curl python-setuptools etc.

(or put them all on separate lines, but it looks weird having a couple on their own lines and four combined here)